pkg/context/httpcontext.go

```diff
@@ -206,10 +249,40 @@ func (ctx *httpContext) Span() tracing.Span {
 	return ctx.span
 }
 
+// Fill the empty tag. If all existing Tags are filled, create new one.
 func (ctx *httpContext) AddTag(tag string) {
 	ctx.tags = append(ctx.tags, tag)
 }
 
+// Add new LazyTag.
+func (ctx *httpContext) AddLazyTag(ns string, prefix string, msg string, intMsg int) {
```

Comment:
the meaning and relationship of the parameters are unclear, could we change the parameter to a closure function (like what we do in `logger.LazyDebug`)